test/paymentmethods.collective.to.collective.test.js

```diff
@@ -0,0 +1,438 @@
+    before('create Collective 2', () => models.Collective.create({ name: "Collective 2", currency: "USD", HostCollectiveId: host1.id, isActive: true }).then(c => collective2 = c));
+    before('create Collective 3', () => models.Collective.create({ name: "Collective 3", currency: "USD", HostCollectiveId: host2.id, isActive: true }).then(c => collective3 = c));
+    // before('Add User 1 as Admin of Collective 1', () => collective1.editMembers([{ id: user1.id, MemberCollectiveId: user1.CollectiveId, role: 'ADMIN' }]));
+    before('create an organization', () => models.Collective.create({ name: "pubnub", currency: "USD" }).then(o => organization = o));
+    before('create a payment method', async () => models.PaymentMethod.create({
+      name: '4242',
+      service: 'stripe',
+      type: 'creditcard',
+      token: 'tok_123456781234567812345678',
+      CollectiveId: organization.id,
+      monthlyLimitPerMember: 10000
+    }).then(pm => stripePaymentMethod = pm));
+    beforeEach('create many transactions', async () => {
+      transactions = [
+        { amount: -500, netAmountInCollectiveCurrency: -500 },
+        { amount: -200, netAmountInCollectiveCurrency: -200 },
+        { amount: -1000,netAmountInCollectiveCurrency: -1000 }
+      ];
+      const transactionsDefaultValue = {
+        CreatedByUserId: user1.id,
+        FromCollectiveId: collective1.id,
+        CollectiveId: organization.id,
+        PaymentMethodId: stripePaymentMethod.id,
+        currency: collective1.currency,
+        HostCollectiveId: collective1.HostCollectiveId,
+        type: 'DEBIT'
+      };
+      await models.Transaction.createMany(transactions, transactionsDefaultValue);
+      await models.Transaction.createManyDoubleEntry(transactions, transactionsDefaultValue);
+    });
+    beforeEach(() => {
+      sandbox = sinon.createSandbox();
+      // And given that the endpoint for creating customers on Stripe
+      // is patched
+      utils.stubStripeCreate(sandbox, { charge: { currency: 'usd', status: 'succeeded' } });
+      // And given the stripe stuff that depends on values in the
+      // order struct is patch. It's here and not on each test because
+      // the `totalAmount' field doesn't change throught the tests.
+      utils.stubStripeBalance(sandbox, ORDER_TOTAL_AMOUNT, 'usd',
+                              0,
+                              STRIPE_FEE_STUBBED_VALUE); // This is the payment processor fee.
+    });
+
+    afterEach(() => sandbox.restore());
+
+
+    it('collective balance for Admin Member matches sum of transactions amounts', async () => {
+      // getting balance of transactions that were just created
+      const reducer = (accumulator, currentValue) => accumulator + currentValue;
+
+      // multiplying by -1 to get the Collective 1 Credit values
+      const balance = (-1) * transactions.map(t => t.netAmountInCollectiveCurrency).reduce(reducer, 0);
```

Comment:
If the transactions above are now positive, you won't need this `-1` here (which was also hard to understand at first)